src/main/java/emu/grasscutter/scripts/ScriptLib.java

```diff
@@ -1013,22 +1013,22 @@ public int ChangeToTargetLevelTag(int var1){
     }
 
     public int AddSceneTag(int sceneId, int sceneTagId){
-        logger.warn("[LUA] Call unimplemented AddSceneTag with {}, {}", sceneId, sceneTagId);
-        //TODO implement
+        logger.debug("[LUA] Call AddSceneTag with {}, {}", sceneId, sceneTagId);
+        sceneScriptManager.get().getScene().getHost().getProgressManager().addSceneTag(sceneId, sceneTagId);
```

Comment:
there is probably a better way of getting the scene or the host

Reply:
I looked at other methods but it seemed like this was going to be the cleanest method from within ScriptLib. If you'd prefer new imports & methods I'm open to suggestion.